test/features/steps/utils.py

```diff
@@ -25,7 +25,7 @@ def run_command_local_or_remote(context, cmd, addr, err_record=False):
         return out
     else:
         try:
-            results = parallax.parallax_call([addr], cmd)
+            results = parallax.parallax_call([addr], cmd, use_crmsh_key=False)
```

Comment:
You shouldn't be using this `parallax.parallax_call` here. You are using the code under test to run tests. 
If you need to use parallax for any reason, import parallax directly and implement the test code.

Reply:
Thanks! changed as using parallax directly in behave functional testing